File: Assignment5/program3.c
///////////////////////////////////////////////////////////////////////////////////
//
//    File namev:  program3.c
//    Description: Accept the number from the user and print the number line
//    Author :     Mayuresh Kaluskar
//    Date :       14/05/2025
//
////////////////////////////////////////////////////////////////////////////////////

#include <stdio.h>

void Display(int iNo)
{
    int iCnt = 0;
     

    for(int iCnt = iNo;iCnt>=(-iNo);iCnt--)
    {
        printf("%d\t",iCnt);
    }


}

int main()
{
    int iVal = 0;

    printf("Enter the number :");
    scanf("%d",&iVal);

    Display(iVal);

    return 0;

}

File: Assignment11/program3.java
import java.util.Scanner;

class Pattern
{
    public void Display(int iNo)
    {
      
      for(int i = 1;i<=iNo;i++)
      {
          System.out.print(i+"\t*\t");
      }
    }
}

class program3
{
    public static void main(String[] args) {
        int iVal = 0;

        Scanner sobj =new Scanner(System.in);

        System.out.println("Enter the number");
        iVal = sobj.nextInt();
         
        Pattern pobj = new Pattern();

        pobj.Display(iVal);
        
    }
}